### ISSUE TITLE
is there any sdk that  can create transaction and sign hex without rpc api?

### ISSUE THREAD

**Issue by zhp1254** [emesik/monero-python#40]
is there any sdk that  can create transaction and sign hex without rpc api?

**Comment by emesik** [comment]
At the moment we have no code that would modify or create transactions without calling the external API.

It is planned for the future but I wouldn't expect it soon. The matter is critical for user's security and needs to be addressed carefully. I'd rather depend on RPC than allow any lousy code there.

**Comment by ruanqunfeng** [comment]
@zhp1254 Do you find the sdk?

**Comment by moneroexamples** [comment]
> is there any sdk that can create transaction and sign hex without rpc api?

Yes there is, but depends what you want to do. If you are after JavaScript tools, look at [mymonero](https://github.com/mymonero/mymonero-core-js) and [openmonero](https://github.com/moneroexamples/openmonero). Openmonero is using mymonero code, so mymonero is the source repo.

**Comment by Monero-HackerIndustrial** [comment]
Is there any update? 
I am working on monerosigner (a seedsigner fork for monero). I wanted to see if there was any traction to have a pure python signing utility. 
Having pure python signing capabilities can help improve development in offline applications
